requirements.txt

```diff
@@ -1,6 +1,13 @@
-numpy~=1.21.4
-open3d~=0.14.1
-PyOpenGL~=3.1.5
-PyQt5~=5.14.1
-pytest~=7.1.1
-pytest-qt~=4.0.2
\ No newline at end of file
+numpy~=1.23.2
+open3d~=0.15.2
```

Comment:
it seems there are no mac/ windows distribution for 0.15.2, see: https://pypi.org/project/open3d/#files
also it seems they remove the 0.15.2 release: https://github.com/isl-org/Open3D/releases/tag/v0.15.1
Can we bump to only 0.15.1?

Reply:
Agreed.

- [ ] decrease open3d version